qlight/config.go

```diff
@@ -0,0 +1,101 @@
+package qlight
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/plugin/security"
+)
+
+type TLSConfig struct {
+	CACertFileName       string
+	ClientCACertFileName string
+	ClientAuth           int
+	CertFileName         string
+	KeyFileName          string
+	InsecureSkipVerify   bool
+	CipherSuites         string
+	ServerName           string
+}
+
+func ConfigureQLightTLSConfig(config *TLSConfig) (*tls.Config, error) {
```

Comment:
In Go they propose to use the package naming as part of the meaning: `qlight` is redundant.

Also with `Configure`, I would changed it to `New` or nothing, as the function returns a new `tls.Config`.

```suggestion
func NewTLSConfig(config *TLSConfig) (*tls.Config, error) {
```

Reply:
suggestion applied